theme/_layouts/default.html

```diff
@@ -54,7 +54,7 @@
 											{% endif %}
 											<div class="feature-list">
 												{% for item in category.subs %}
-													{% if page.url contains item.url %}
```

Comment:
To keep the logical order or the contains check I'd suggest to update the changes to at all occurrences:
```
{% assign cleanItemUrl = item.url | split: '?' | first %}
{% if page.url contains cleanItemUrl %}
```

Reply:
Done.